src/astro/databases/snowflake.py

```diff
@@ -20,6 +34,11 @@ class SnowflakeDatabase(BaseDatabase):
     """
 
     def __init__(self, conn_id: str = DEFAULT_CONN_ID):
+        self.native_support: Dict[Any, str] = {
+            FileLocation.GS: "gs_to_snowflake",
+            FileLocation.S3: "s3_to_snowflake",
+        }
+        self.integration: Optional[str] = None
```

Comment:
It may be worth naming it `storage_integration` and adding a link to some Snowflake documentation about it, e.g.:
https://docs.snowflake.com/en/sql-reference/sql/create-storage-integration.html

Reply:
Done